심화요구사항.md

```
@@ -1,29 +1,16 @@
-  repository:
-    type: jcf   # jcf | file
-```
+> [화면 가이드](https://www.notion.so/18f6fd228e8d80828264ca0b1fb4078f?pvs=21)
 
-- [x] discodeit.repository.type 설정값에 따라 Repository 구현체가 정해집니다.
-    - [x] 값이 jcf 이거나 없으면 JCF*Repository 구현체가 Bean으로 등록되어야 합니다.
-    - [x] 값이 file 이면 File*Repository 구현체가 Bean으로 등록되어야 합니다.
+- [ ]  [Railway.app](http://railway.app/)을 활용하여 애플리케이션을 배포해보세요.
 
----
+    - Railway.app은 애플리케이션을 쉽게 배포할 수 있도록 도와주는 PaaS입니다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)